presentation/presentation.go

```diff
@@ -712,6 +722,12 @@ func (p *Presentation) AddImage(i common.Image) (common.ImageRef, error) {
 	if i.Size.X == 0 || i.Size.Y == 0 {
 		return r, errors.New("image must have a valid size")
 	}
+	if i.Path != "" {
+		err := tempstorage.Add(i.Path)
```

Comment:
when does this get cleaned up?

Reply:
Add is working only for memstore, it doesn't create files on disk. Do we need to clean it up in memory?

Reply:
Yes it would be good to unreference it, then it should be cleaned up by GC.
Why is this only necessary when working with in memory?

Reply:
because the file which is added from Add will be read in the future. in the case of disk storage the file is already on disk, so unioffice will find it there by path, and for memstorage we should add it separately, otherwise unioffice will not find it

Reply:
OK.  Is there any way to clean it up at the end, like when the document/presentation is closed/finalized?

Reply:
yes, I will add finalizers

Reply:
Probably can clear the storage after write?  Just need to be careful if there are multiple documents processing simultaneously

Reply:
clean storage after write to disk?